docs/reference/codelists.md

```diff
@@ -112,11 +93,37 @@ file: ../../codelists/closed/geometry_type.csv
 ---
 ```
 
-### risk_data_type
+### hazard_type
+
+The RDLS offers a classification of hazards that are more often required in disaster risk assessments, based on the review and mapping of existing alternative definitions into one consistent framework.
```

Comment:
Just a question about this bit, do we need to write explanations for all of the codelists and include them in this?

Reply:
I don't think so since the codelist are linked from the schema tables, which provide the context of what they are used for. I just copied this text from the old `hazard.md` page in order to preserve it.